modcc/expression.hpp

```diff
@@ -294,7 +294,7 @@ class ARB_LIBMODCC_API IdentifierExpression : public Expression {
     }
 
     expression_ptr clone() const override;
-
+    using Expression::semantic;
```

Comment:
Seems strange that this would be needed - do you have any explanation why only for this specific override the base class' method needs to be pulled into scope?

Reply:
No, this is an entirely correct complaint. The problem is that we have this situation
```cxx
struct Base {
  virtual void foo() {}
  virtual void foo(int) {}
};

struct Derived {
  void foo() override {}
};
```
which is overriding only one overload. The `using` declares we want to use the `Base::foo(int)`
but overload `Base::foo()`.